scapy/contrib/bgp.py

```diff
@@ -12,6 +12,7 @@
 import struct
 import re
 import socket
+from typing import Any
```

Comment:
`from scapy.compat import Any` for the moment. All our typing currently goes through compat